src/Leaflet.Renderer.SVG.Tile.js

```diff
@@ -39,6 +39,16 @@ L.SVG.Tile = L.SVG.extend({
 		}
 	},
 
+	removeFrom: function (map) {
+		if (this.options.interactive) {
+			for (var i in this._layers) {
+				var layer = this._layers[i];
+				delete this._map._targets[L.stamp(layer._path)];
```

Comment:
I have written up #79 with some ideas. By avoiding the `Layer` overhead for vector-tiles and their features, the code should hopefully become more straight-forward and easier to reason about.